<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTaskForm</class>
 <widget class="QWidget" name="EditTaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="notes_text_edit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="CustomComboBox" name="task_dropdown"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="CustomComboBox" name="project_dropdown"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="confirm_button">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button">
       <property name="styleSheet">
        <string notr="true">background-color: red</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="time_worked_line"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>customcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
